### PR TITLE
Display notification when a newer version is available

### PR DESCRIPTION
**Pull request by astronomersiva**
Addresses #326

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -14,6 +15,12 @@ const mergePackageJson = require('merge-package.json');
 const gitDiffApply = require('git-diff-apply');
 const run = require('./run');
 const utils = require('./utils');
+const pkg = require('../package.json');
+
+updateNotifier({
```
Comment (kellyselden) [inline src/index.js]: Can we move this to the bin/ember-cli-update.js file please? We probably shouldn't print this if used as a library vs an executable.

Comment (astronomersiva) [reply src/index.js]: That makes sense, will make the change.

**Comment by astronomersiva** [comment]
@kellyselden I have moved the notifier to bin/ember-cli-update as suggested..

**Comment by kellyselden** [comment]
Do you think you could rebase to resolve the conflicts before merging?

**Comment by kellyselden** [comment]
Perfect. Thanks!

**Comment by astronomersiva** [comment]
@kellyselden, we could also leave a note in the README about using this with ‘npx’. It will always run the latest version.

**Comment by kellyselden** [comment]
Sounds good to me

**Pull request merged**